openlibrary/templates/books/custom_carousel.html

```diff
@@ -8,15 +8,21 @@ <h2 class="home-h2"><a href="$url">$title</a></h2>
         </div>
       <div class="carousel-container carousel-container-decorated">
         <div class="carousel carousel-$key">
-          $for book in (books or []):
+          $for index, book in enumerate(books or []):
             $ byline = (' by ' + ', '.join([author.name for author in book.authors])) if book.get('authors') else ''
             $ byline_links = (' by ' + ', '.join(['<a href="%s">%s</a>' % (author.key, author.name) for author in book.authors])) if book.get('authors') else ''
+            $if index > 5:
```

Comment:
We want this to work for non-js users (the site already works pretty well for them) and be indexed by search engine. Right now at maximum 5 can be shown without scrolling (in future we could hide all the others if necessary).

happy to use config if that's a better way to do this but I don't think we should load them all via JS.

Reply:
naming the magic number would help
You can also add a comment above it with:
 `$# this syntax`

Reply:
The advantage to lazy loading everything is that it would also deal with below the fold images. The home page is VERY long and only a tiny fraction of it is visible.

Reply:
Sure but it's a trade off. You break the experience for older browsers and users without javascript. When you scroll content becomes visible.

The main issue with the existing setup is we make 100 img requests on load. Getting that down to a manageable amount should be priority 1. 

After that I'm not convinced lazy loading the rest will make much of a performance difference, but we can consider the value of that (and maybe prove me wrong) separately.